openerp/addons/base/res/res_config.py

```diff
@@ -437,7 +437,7 @@ def onchange_module(self, field_value, module_name):
 
         if modules and not field_value:
             dep_ids = modules.downstream_dependencies()
-            dep_name = (ModuleSudo.browse(dep_ids) + modules).mapped('shortdesc')
+            dep_name = (dep_ids|modules).mapped('shortdesc')
```

Comment:
Just a comment about Python style: could you add spaces around the `|` operator? Thanks!